pkg/dashboard/dashboard.go

```diff
@@ -162,7 +164,12 @@ func GetRouter(port int, basePath string, vpaLabels map[string]string, excludeCo
 			return
 		}
 
-		data, err := summary.GetInstance().Run(vpaLabels, excludeContainers)
+		summarizer := summary.NewSummarizer(
```

Comment:
If we want to make use of the caching functionality in the `Summarizer` type, the `Summarizer` object should get passed into `GetRouter` from `cmd/dashboard.go`

Reply:
Absolutely! This is a shim for now so that the commits touched a more isolated package at one time. Next up is the dashboard updates (which also includes a revisit to the Summarizer package to assist the dashboard get the information it wants).